openmetadata-ui/src/main/resources/ui/src/components/TasksDAGView/TasksDAGView.tsx

```diff
@@ -125,8 +144,17 @@ const TasksDAGView = ({ tasks, selectedExec }: Props) => {
       onInit={(reactFlowInstance) => {
         onLoad(reactFlowInstance);
       }}
-      onNodesChange={onNodesChange}
-    />
+      onMove={(_e, viewPort) => handleZoomLevel(viewPort.zoom)}
+      onNodesChange={onNodesChange}>
+      <CustomTasksDAGViewControl
+        fitViewParams={{
+          minZoom: MIN_ZOOM_VALUE,
+          maxZoom: MAX_ZOOM_VALUE,
+        }}
```

Comment:
Try to avoid create a object and pass it as prop in the return statement as it will be treated as new prop for child